tools/make/kube.mk

```diff
@@ -96,6 +96,25 @@ kube-demo-undeploy: ## Uninstall the Kubernetes resources installed from the `ma
 .PHONY: conformance
 conformance: create-cluster kube-install-image kube-deploy run-conformance delete-cluster ## Create a kind cluster, deploy EG into it, run Gateway API conformance, and clean up.
 
+.PHONY: e2e
+e2e: create-cluster kube-install-image kube-deploy install-ratelimit run-e2e delete-cluster
+
+.PHONY: install-ratelimit
+install-ratelimit:
+	@$(LOG_TARGET)
+	kubectl apply -f examples/redis/redis.yaml
+	kubectl rollout restart deployment envoy-gateway -n envoy-gateway-system
+	kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
+
+.PHONY: run-e2e
```

Comment:
How about `run-e2e-ratelimit` ? Since we might need to add other e2e tests like auth, etc.

Reply:
this's the entrypoint for all e2e tests, for now there's only ratelimit tests